chipsec/module_common.py

```diff
@@ -132,7 +132,9 @@ def getReturnCode(self, result: int) -> int:
 
 def getModuleResultName(res) -> str:
     if chipsec.chipset.cs().using_return_codes:
-        return 'Passed' if (res & 0xFFFFFFFF00000000) == 0 else 'Failed'
+        result_mask = 0xFFFFFFFF00000000
+        status = [0, 1, 256]
```

Comment:
Can we use `[ModuleResult.status.SUCCESS[0], ModuleResult.status.INFORMATION[0], ModuleResult.status.INVALID[0]]` instead of `[0, 1, 256]`?